### PR TITLE
🌐 Add Chinese translations for `docs/zh/docs/advanced/websockets.md`

### PR DESCRIPTION
**Pull request by ChuyuChoyeon**
For your approval

### CODE REVIEW

**Comment by tiangolo** [comment]
📝 Docs preview for commit fc72a53116e52b5702403f8292245d0cd0b4b2ea at: https://6484f4be6038ce29594f6304--fastapi.netlify.app

**Comment by ChoyeonChern** [comment]
@tiangolo

**Review by Chushine — APPROVED** [review]

**Review by Chushine — APPROVED** [review]

**Comment by tiangolo** [comment]
📝 Docs preview for commit c0ac458568a203a932c6d66fe9ea3dcb1d7a3e9e at: https://648bd8ce63785e205c0ac1e4--fastapi.netlify.app

File: docs/zh/docs/advanced/websockets.md [hunk 1]
```diff
@@ -0,0 +1,216 @@
...
+
+## WebSockets 客户端
+
+### 在生产环境中
+
+在您的生产系统中，您可能使用现代框架（如React、Vue.js或Angular）创建了一个前端。
+
+要使用 WebSockets 与后端进行通信，您可能会使用前端的工具。
+
+或者，您可能有一个本机移动应用程序，直接使用本机代码与 WebSocket 后端通信。
```
Comment (wdh99) [inline docs/zh/docs/advanced/websockets.md:23]: native mobile application -> 原生移动应用. (移动开发中有三个名字Native, Web, Hybrid——原生，web，混合。这里的native应该就是指原生)
native code  -> （所以这里应该也是“原生”的意思）原生代码

Comment (ChoyeonChern) [reply docs/zh/docs/advanced/websockets.md]: 已经修改了

**Comment by tiangolo** [comment]
📝 Docs preview for commit a102199668444f6c1fd9af01208fc2ed76c29ee6 at: https://648c5524ce5abc0979ea7f6c--fastapi.netlify.app

**Review by wdh99 — APPROVED** [review]

**Comment by tiangolo** [comment]
📝 Docs preview for commit ef5eb0e3c04bc5eb085e58aa49478a694b71071b at: https://648c5b8dc985170d2f9b315c--fastapi.netlify.app

**Pull request merged**